src/main/java/com/mewna/catnip/rest/ResponsePayload.java

```diff
@@ -27,26 +27,35 @@
 
 package com.mewna.catnip.rest;
 
-import io.vertx.core.buffer.Buffer;
-import io.vertx.core.json.JsonArray;
-import io.vertx.core.json.JsonObject;
+import com.grack.nanojson.JsonArray;
+import com.grack.nanojson.JsonObject;
+import com.grack.nanojson.JsonParser;
+import com.grack.nanojson.JsonParserException;
 
 public class ResponsePayload {
-    private final Buffer buffer;
```

Comment:
Did you just replace all uses of Buffer with String?

Reply:
only this one